polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -2421,17 +2422,16 @@ impl<T: Config> Pallet<T> {
 		mut message: Xcm<()>,
 	) -> Result<XcmHash, SendError> {
 		let interior = interior.into();
+		let origin_dest = interior.clone().into();
```

Comment:
This is the `origin` as seen by the local chain, relative to `Here`.

```suggestion
		let local_origin = interior.clone().into();
```